Source/Diagnostics/Diagnostics.cpp

```diff
@@ -243,7 +243,114 @@ Diagnostics::InitBaseData ()
         m_flush_format = std::make_unique<FlushFormatCheckpoint>() ;
     } else if (m_format == "ascent"){
         m_flush_format = std::make_unique<FlushFormatAscent>();
-    } else if (m_format == "sensei"){
+        std::cout<< " m_diag_name " <<m_diag_name<<std::endl;
+        amrex::ParmParse pp_diag_viz(m_diag_name);
+        int write_yaml_file = 0;
+        pp_diag_viz.query("viz", write_yaml_file);
+        if (write_yaml_file){
+          std::ofstream yaml_file("ascent_actions.yaml");
+          // This block is data pipelines
+          // Pipelines allow users to compose filters that transform the published input data into new meshes.
+          // Each entry in pipelines: executes a series of functions from top to bottom,
+          // Results of prior functions can be used in later calls of the same pipeline
+          yaml_file<<"#   This block are data pipelines"<<std::endl;
```

Comment:
Although it's less pretty, let us use `\n` for newlines.
Every `std::endl` implies a `yaml_file.flush()` and doing thos at the end (on close) is more than sufficient.

https://en.cppreference.com/w/cpp/io/manip/endl

Reply:
Done. 

Thanks!